{"name": "Принять заказ без id заказа", "status": "failed", "statusDetails": {"message": "AssertionError: assert (404 == 400)\n +  where 404 = <Response [404]>.status_code", "trace": "self = <tests.test_orders_accept.TestOrdersAccept object at 0x000001B060532420>, create_and_delete_courier = (<Response [201]>, {'login': 'rqwaiwwbvf', 'password': 'gmtilyhkeo'}, {'login': 'rqwaiwwbvf', 'password': 'gmtilyhkeo'})\n\n    @allure.title('Принять заказ без id заказа')\n    @allure.description('Попытка принять заказ без id заказа должна вернуть ошибку 400')\n    def test_accept_order_without_order_id_fail(self, create_and_delete_courier):\n        r = requests.post(TestCourierLinks.login_url, data=create_and_delete_courier[2])\n        courier_id = (r.json()['id'])\n        params = {'courierId': courier_id}\n        response = requests.put(OrdersLinks.accept_order, params=params)\n>       assert response.status_code == 400 and response.json()['message'] == \"Недостаточно данных для поиска\"\nE       assert (404 == 400)\nE        +  where 404 = <Response [404]>.status_code\n\ntests\\test_orders_accept.py:55: AssertionError"}, "description": "Попытка принять заказ без id заказа должна вернуть ошибку 400", "start": 1701451458841, "stop": 1701451459374, "uuid": "2f2ca3a4-edf4-412c-a277-3bfdf041ef15", "historyId": "bad7bfe9659a94cf4e8767c4f6060d28", "testCaseId": "bad7bfe9659a94cf4e8767c4f6060d28", "fullName": "tests.test_orders_accept.TestOrdersAccept#test_accept_order_without_order_id_fail", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_orders_accept"}, {"name": "subSuite", "value": "TestOrdersAccept"}, {"name": "host", "value": "Ya"}, {"name": "thread", "value": "34240-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_orders_accept"}]}